firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -75,6 +78,7 @@ protected void onCreate(Bundle savedInstanceState) {
     if (savedInstanceState != null) {
       releaseName = savedInstanceState.getString(RELEASE_NAME_KEY);
       infoText = savedInstanceState.getCharSequence(INFO_TEXT_KEY);
+      feedbackTrigger = FeedbackTrigger.valueOf(savedInstanceState.getString(FEEDBACK_TRIGGER_KEY));
```

Comment:
I think this might not work as intended. I think the default `valueOf()` implementation is based on the `name()` of the `Enum`, not the customized `toString()` that you implemented. So you could implement a `fromString()` and use that instead of `valueOf()` here - or use `name()` instead of `toString()` when serializing the value for the `Bundle`.